src/scripts/main.js

```diff
@@ -1 +1,20 @@
 'use strict';
+
+const gallery = document.querySelector('.gallery');
+
+gallery.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const link = e.target.closest('.list-item__link');
+  const targetPlace = document.querySelector('.gallery__large-img');
+
+  if (!link) {
+    return;
+  }
+
+  const targetImg = link.querySelector('.gallery__thumb');
```

Comment:
Ensure that the `.gallery__thumb` element is indeed a child of `.list-item__link` in your HTML structure. If not, this query will return `null`, and the image source won't be updated as expected.